WordPress/gutenberg
# Make strings in theme.json translatable

## What?

[Aspect Ratios](https://github.com/WordPress/gutenberg/blob/trunk/lib/theme.json#L195-L231) and [Shadow Presets](https://github.com/WordPress/gutenberg/blob/trunk/lib/theme.json#L235-L261) are untranslated. The strings [aren't in GlotPress](https://translate.wordpress.org/projects/wp/dev/ro/default/?filters%5Bterm%5D=Classic+Portrait+-+2%3A3&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=asc#).
## Why?

This PR resolves the missing Aspect Ratios in https://github.com/WordPress/gutenberg/issues/66027.
## How?

By adding a symlink to `theme.json` in the project's root directory, [strings will be extracted](https://github.com/wp-cli/i18n-command/blob/065bb3758fcbff922f1b7a01ab702aab0da79803/src/ThemeJsonExtractor.php#L19)
## Testing Instructions

1. Download the plugin locally.
2. Make POT with `wp i18n make-pot .`.
3. Inspect the generated `gutenberg.pot` file to confirm that the **Aspect Ratio** strings from `theme.json` are not being extracted.

N.B **the `shadow.presets` strings are not extracted.**

### Testing Instructions for Keyboard


## Screenshots or screencast
